src/packages/cli/src/args.ts

```diff
@@ -160,8 +165,9 @@ export default function (
 
   // disable dot-notation because yargs just can't coerce args properly...
   // ...on purpose! https://github.com/yargs/yargs/issues/1021#issuecomment-352324693
-  yargs
+  const yargsParser = yargs()
     .parserConfiguration({ "dot-notation": false })
+    .exitProcess(false)
```

Comment:
By default Yargs terminates the process if `--help` provided, or if validation fails.

With this change, we will pass through if `--help` is provided (and return `{ action: "help }`), and throw if validation fails.

This allows us to test args validation more thoroughly.